.circleci/config.yml

```diff
@@ -102,9 +102,22 @@ jobs:
       - store_test_results:
           path: collect_app/build/test-results
 
+  build_instrumented:
```

Comment:
This was adding a couple of mins onto our `test_app` tasks so made sense to pull it out so it could run in parallel and keep the PR build time down.